observability/network_observability/netobserv_cli/netobserv-cli-install.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="netobserv-cli-install"]
+= Installing the netobserv CLI
+include::_attributes/common-attributes.adoc[]
+:context: cli-install
+
+toc::[]
+
+You can quickly debug and troubleshoot with Network Observability by using the Network Observability `netobserv` CLI. The `netobserv` CLI is a flow and packet visualization tool that relies on the eBPF agent to stream collected data to a local collector. It requires no storage, compared with the Operator version of Network Observability, since each node generates a file containing its related data. This enables quick, live insight into packets and flow data and requires no configuration. 
```

Comment:
Some updates here:
- `netobserv` is the upstream name and the command to run. The downstream name of the product is still `Network Observability CLI`
- the files are not separated per node anymore
- there is still some configuration as command parameters to enable features / filters
- the most important note here is that we don't need any installation

```suggestion
You can quickly debug and troubleshoot with Network Observability by using the Network Observability CLI. The CLI is a flow and packet visualization tool that relies on the eBPF agents to stream collected data to an ephemeral collector pod. It requires no persistent storage during the capture. After the run, the output is transfered to your local machine. This enables quick, live insight into packets and flow data without installing any operator. 
```

Reply:
Thanks! 
- I was thinking that we would refer to the Network Observability CLI like how we refer to the [OpenShift CLI (`oc`) ](https://docs.openshift.com/container-platform/4.15/cli_reference/openshift_cli/getting-started-cli.html), since the command in the terminal is `netobserv`. See how in the link, they said "OpenShift CLI (`oc`)"
- removed
- gotcha, I think I was meaning that there's no _storage_ configuration, so I clarified this bit. 
- Okay, I included that the Operator doesn't need to be installed in the Procedure module, `/modules/network-observability-netobserv-cli-install.adoc`

Reply:
Going back through my comments here, and there was a little more I needed to address, so I added in your suggested text as well.